packages/models-library/tests/test_project_nodes_io.py

```diff
@@ -180,3 +182,21 @@ def test_simcore_s3_directory_get_parent():
         SimcoreS3DirectoryID._get_parent(  # noqa SLF001
             "/hello/object/", parent_index=4
         )
+
+
+USER_ID_0: UserID = 0
+
+
+@pytest.mark.parametrize(
+    "object_key",
+    [
+        f"api/{UUID_0}/some-random-file.png",
+        f"exports/{USER_ID_0}/some-random-file.png",
```

Comment:
what do you really expect as naming here? is this real? a random name? or is it a UUID?
if not a UUID how is the collision going to be handled?